include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -227,6 +227,67 @@ def TTG_InsertSliceOp : TTG_Op<"insert_slice",
 }
 
 
+def TTG_ViewSliceOp : TTG_Op<"view_slice",
```

Comment:
I retained syntax of the extract_slice operation, so this td description is (almost) the same as for extract_slice.